soc/arm/nuvoton_npcx/common/reg/reg_def.h

```diff
@@ -21,6 +21,24 @@
 	BUILD_ASSERT(offsetof(struct reg_def, member) == offset, \
 		"Failed in offset check of register structure member!")
 
+/*
+ * NPCX register access checking via structure macro function to mitigate the
+ * risk of unexpected compiling results if module contains different length
+ * registers. For example, a word register access might break into two byte
+ * register accesses by adding 'packed' attribute.
+ */
+#define NPCX_REG_WORD_ACCESS_CHECK(reg, val) { \
```

Comment:
Please add a comment for each macro so we know what the args are supposed to me and what it does

Reply:
Done.